cli/src/template_builder.rs

```diff
@@ -161,6 +167,12 @@ where
     Self: WrapTemplateProperty<'a, Timestamp>,
     Self: WrapTemplateProperty<'a, TimestampRange>,
 {
+    // Don't write default implementations on these (and certainly not ones
+    // using another function on self): wrapper types that contain a sum type
+    // of a CoreTemplatePropertyKind or a custom type will hit the default case
+    // every time rather than the implementation on CoreTemplatePropertyKind if
+    // a custom implementation is omitted.
```

Comment:
nit: I wouldn't add this comment because I don't think it is any special here. We shouldn't add default implementation unless that makes sense.